### ISSUE TITLE
[FR] Content Analytics

### ISSUE THREAD

**Issue by khalwat** [nystudio107/craft-seomatic#10]
The Content Analytics sidebar doesn't show anything interesting yet

**Comment by LouisCuvelier** [comment]
If you do something like [Yoast does on WordPress](https://yoast.com/research/real-time-content-analysis/), you have a buyer !

![image](https://user-images.githubusercontent.com/34551650/79920110-fa8b8b80-842f-11ea-9fa6-b78193788cd9.png)

**Comment by khalwat** [comment]
Yep, that's the idea -- it'd be much more comprehensive, though.

**Comment by LouisCuvelier** [comment]
Do you have an ETA ?

**Comment by khalwat** [comment]
No current ETA, but hopefully by Q3 2020.

**Comment by masonry47** [comment]
Hi there!  Any update on an ETA on this feature?  Thanks!

**Comment by development-lameco** [comment]
Found this FR after looking into the old SEOMatic 2 repo mentioning something about SEOMetrics.
+1 on this FR. Clients are asking us about this feature and even referring to the https://github.com/ethercreative/seo plugin.
Although I don't think we could use both plugins side-by-side I also prefer SEOmatic over Ether SEO because it has so much more features build in already.

**Comment by khalwat** [comment]
Noted @development-lameco !

**Comment by stigvanbrabant** [comment]
Has something like this been implemented yet?
